### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by uzzi1771**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/uzzi1771/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (uzzi1771) [inline ReadMe.md:1]: !fixed

Comment (uzzi1771) [inline ReadMe.md:1]: !fixed

Comment (uzzi1771) [inline ReadMe.md:1]: !fix tyi

Comment (uzzi1771) [inline ReadMe.md:1]: /cf map-route $A cfapps.io -n review-ninja

**Pull request merged**